### ISSUE TITLE
Only include blockers in dependency chains on FixVersion reports

### ISSUE THREAD

**Issue by jillichrome** [riptano/argus#81]
josh-mckenzie: Instead of showing every ticket related to everything else, we need to define a subset of relationships to transitively apply FixVersion to. For example, if a ticket is flagged release 1.0 and is related to another ticket, that other ticket should not be shown. If a ticket is instead flagged release 1.0 and is blocked by another ticket, that other ticket should be shown.
